app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  href ||= false
+  image_src ||= false
+  image_alt ||= ""
+  heading_text ||= false
+  metadata ||= false
```

Comment:
Is this the same concept as `context` in the [title component](https://github.com/alphagov/static/blob/master/app/views/govuk_component/title.raw.html.erb#L3)?

Reply:
I think so, will change.